pkg/tracing/provider/jaeger/jaeger.go

```diff
@@ -0,0 +1,59 @@
+package jaeger
+
+import (
+	"context"
+	"io"
+
+	"github.com/go-kit/kit/log"
+	"github.com/improbable-eng/thanos/pkg/tracing"
+	"github.com/opentracing/opentracing-go"
+	"github.com/uber/jaeger-client-go"
+	"github.com/uber/jaeger-client-go/config"
+	"github.com/uber/jaeger-lib/metrics/prometheus"
+	"gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	ServiceName string `yaml:"service_name"`
+}
+
+func parseConfig(conf []byte) (Config, error) {
+	config := Config{}
```

Comment:
Too shallow function IMO - it does not hide much. What do you think about inlining this?